### ISSUE TITLE
Copy-paste error in the description of `validate`

### ISSUE THREAD

**Issue by ryzhyk** [TimelyDataflow/differential-dataflow#181]
I think this comment was copy-pasted from `propose.rs`:

https://github.com/TimelyDataflow/differential-dataflow/blob/92134466b01f799757c50f693a222a53f2c38f59/dogsdogsdogs/src/operators/validate.rs#L12

**Comment by frankmcsherry** [comment]
That's absolutely correct. I .. suspect I was being lazy with the whole PR and was hoping to get a work/doesn't bit out before documenting it, and then failed to pay attention to whether it worked. :)

Let's leave the issue open for "tidy up documentation, examples, code".
